docs/examples/qsplot.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+import cartopy.crs as ccrs
+import flopy
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+from matplotlib.collections import PolyCollection
+
+QS_ROOT = os.path.abspath(os.path.join(__file__, ".."))
+modelname = "mymodel"
+ws = os.path.join(QS_ROOT, "quickstart_data")
+
+
+def _pcolormesh(grid, da, ax):
+    vertices = []
+    for face in grid.iverts:
+        f = []
+        for i in face:
+            f.append(grid.verts[i])
+        vertices.append(f)
+    vertices = np.array(vertices)
+
+    collection = PolyCollection(vertices)
+    collection.set_array(da.to_numpy().ravel())
+    p = ax.add_collection(collection, autolim=False)
+
+    xmin, xmax, ymin, ymax = grid.extent
+    ax.set_xlim(xmin, xmax)
+    ax.set_ylim(ymin, ymax)
+    cverts = (xmin, ymin), (xmax, ymax)
+    ax.update_datalim(cverts)
+
+    return p
+
+
+# create budget reader
+bpth = os.path.join(ws, f"{modelname}.bud")
+bobj = flopy.utils.CellBudgetFile(bpth, precision="double")
```

Comment:
I guess @deltamarnix plans to see if we can swap this and line 46 out with (lazy) output loading routines